hibernate-search-plugins/org.jboss.tools.hibernate.search/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,30 @@
+Manifest-Version: 1.0

```

Comment:
@Sanne no, it's no generated. It's a standard [OSGi bundle manifest file](https://help.eclipse.org/neon/index.jsp?topic=%2Forg.eclipse.platform.doc.isv%2Freference%2Fmisc%2Fbundle_manifest.html). 
About the version - during the maven build tycho checks if versions match each other: https://wiki.eclipse.org/Tycho/Packaging_Types